AdobeStockImage/Model/ImageSeriesSerialize.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Model;
+
+use Magento\AdobeStockAssetApi\Api\Data\AssetInterface;
+use Magento\Framework\Exception\SerializationException;
+
+/**
+ * Class ImageSeriesSerialize
+ */
+class ImageSeriesSerialize
+{
+    /**
+     * Serialize image series data.
+     *
+     * @param AssetInterface[] $series
+     *
+     * @return array
+     * @throws SerializationException
+     */
+    public function execute(array $series): array
```

Comment:
I'd propose to use the existing hydrator to extract data from `DocumentInterface`